UA-CloudLibrary.sln

```diff
@@ -22,8 +22,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SampleConsoleClient", "Samp
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Opc.Ua.CloudLib.Client", "Opc.Ua.CloudLib.Client\Opc.Ua.CloudLib.Client.csproj", "{8353C72D-E29A-40BE-A3EF-C59330FC270C}"
 EndProject
-Project("{E53339B2-1760-4266-BCC7-CA923CBCF16C}") = "docker-compose", "docker-compose.dcproj", "{C7282E18-BCDA-48F6-99FD-EC7142A0397A}"
```

Comment:
i vote again for to keep the docker-compose project in the solution ;)

Reply:
:-)